src/WebJobs.Script/Workers/Http/HttpWorkerConstants.cs

```diff
@@ -13,5 +13,8 @@ public static class HttpWorkerConstants
         // Child Process Env vars
         public const string PortEnvVarName = "FUNCTIONS_HTTPWORKER_PORT";
         public const string WorkerIdEnvVarName = "FUNCTIONS_HTTPWORKER_ID";
+        public const string FunctionAppRootVarName = "FUNCTIONS_APP_ROOT_PATH";
+        public const string CustomHandlerPortEnvVarName = "FUNCTIONS_CUSTOMHANDLER_PORT";
+        public const string CustomHandlerWorkerIdEnvVarName = "FUNCTIONS_CUSTOMHANDLER_WORKER_ID";
```

Comment:
@anthonychu - Take a look at the environment variable names.

Reply:
Will we continue to populate `FUNCTIONS_HTTPWORKER_PORT` and `FUNCTIONS_CUSTOMHANDLER_PORT`? (either until we remove the `httpWorker` section or maybe forever just to avoid breaking existing apps)

Reply:
we can leave them.